CHANGELOG.md

```diff
@@ -11,4 +11,7 @@
 [Unreleased](https://github.com/adobe/xdm/releases/tag/v0.9.1)
 
 * Renames all `Metric` schemas to `Metric Definition` #254
+* Moved `Commerce` metrics used in ExperienceEvent `metrics` to the `commerce` context #317
+* Shortened the URI field name for the web events to `https://ns.adobe.com/xdm/metrics/*` by dropping the 'data'. #317
```

Comment:
This change is incomplete (you also need to create a new directory, explain it in all documents that mention the repo structure, adjust all URLs), breaking (lots of stuff) and overall not worth it.